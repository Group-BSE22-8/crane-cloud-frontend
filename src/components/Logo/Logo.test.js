import React from 'react';

import { shallow } from 'enzyme';

import Logo from './index';

describe('Logo component test', () => {
    it('checks whether the component rendered with expected elements', () => {
        const LogoComponent = shallow(<Logo />);
        
        expect(LogoComponent).toMatchSnapshot();

        const LogoType = LogoComponent.find("div > div");
        expect(LogoType.hasClass("LogoType")).toEqual(true);
        expect(LogoType.text()).toBe("Crane Cloud");
    })
})