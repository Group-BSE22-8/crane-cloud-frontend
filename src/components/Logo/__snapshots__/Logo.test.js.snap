// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component test checks whether the component rendered with expected elements 1`] = `ShallowWrapper {}`;
